tests/console/gpg.pm

```diff
@@ -109,7 +109,8 @@ sub gpg_encrypt_file {
 }
 
 sub run {
-    select_console 'user-console';
+    select_console(check_var('ARCH', 's390x') ? 'root-console' : 'user-console');
+    record_soft_failure 'boo#1097610' if check_var('ARCH', 's390x');
```

Comment:
Maybe the relation between `select_console('user-console')` and the `record_soft_failure` would be more visible in the code, if you would put them together in an `if (check_var('ARCH', 's390x') {` block.

Reply:
Good idea, implemented it